Formula/mpage.rb

```diff
@@ -0,0 +1,21 @@
+class Mpage < Formula
+  desc "Mpage, many to one page printing utility"
+  homepage "http://www.mesa.nl/pub/mpage/"
+  url "http://www.mesa.nl/pub/mpage/mpage-$version.tgz"
+  version "2.5.6"
+  sha256 "4fe66dfc27f7c4bfbca60ef617f968aa2e6ee877e8921aa968c16a03aa4edc04"
+
+  def install
+    args = %W[
```

Comment:
The Makefile is rather hilarious:

```Makefile
BINDIR=$(PREFIX)/bin
LIBDIR=$(PREFIX)/share
MANDIR=$(PREFIX)/man/man1
```

Just a comment. You should probably ask upstream to fix it.